src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -122,128 +72,65 @@ Safari and Chrome add a tap highlight color which can be prevented with the
 {% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/states-example.html" region_tag="webkit-specific" adjust_indentation="auto" %}
 </pre>
 
-[Try it](https://googlesamples.github.io/web-fundamentals/fundamentals/design-and-ui/input/touch/states-example.html){: target="_blank" .external }
```

Comment:
Please keep try it buttons unless theres a reason developers shouldn't be able to run the code